siwe/siwe.py

```diff
@@ -82,30 +83,11 @@ def __get_validators__(cls):
         yield cls.validate
 
     @classmethod
-    def validate(cls, v):
-        if not isinstance(v, str):
-            raise TypeError("string required")
+    def validate(cls, v:str):
         cls.date = isoparse(v)
         return cls(v)
 
 
-class Address(str):
```

Comment:
This custom type doesn't add much, and is coupled to the `CheckSumAddress` type anyway through `Web3.is_checksum_address`